src/subscription_manager/gui/registergui.py

```diff
@@ -137,6 +137,51 @@ def pop(self, index=None):
         p = self._list.pop()
         return p
 
+class ServerInfo(object):
+    """Object holding server connection information
+
+    A container class useful for holding the connection information
```

Comment:
(minor) To me, 'container class' implies something like a list/array/tree for holding multiples of some type of object.